docs/en/02_Developer_Guides/06_Testing/00_Unit_Testing.md

```diff
@@ -110,7 +110,7 @@ use SilverStripe\Dev\SapphireTest;
 
 class PageTest extends SapphireTest
 {
-    public function setUp()
+    public function setUp(): void
```

Comment:
This is the kind of thing that would ideally be able to be automated by the upgrader, since it's going to be very common in modules and projects. Relevant issue: https://github.com/silverstripe/silverstripe-upgrader/issues/175